docs/package.json

```diff
@@ -18,6 +18,7 @@
     "@docusaurus/preset-classic": "2.4.1",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
+    "docusaurus": "^1.14.7",
```

Comment:
I don't understand why do you added this extra dependency here? This must be the reason why the lockfile is very different.

Reply:
I didn't add up any dependency separately, I installed docusaurus. And that should be why they are differences.